go/flake.nix

```
@@ -14,6 +14,43 @@
         overlays = [ (final: prev: { go = prev."go_1_${toString goVersion}"; }) ];
         pkgs = import nixpkgs { inherit overlays system; };
 
+        # common dependency
+
+        # custom fork of swagger used by efishery
+        # due to its ability to add custom tag on generated code
+        # the nix package did not expose any param to override
+        # thus we make a separate package for this flake
+        # ref: https://github.com/NixOS/nixpkgs/blob/nixos-23.11/pkgs/development/tools/go-swagger/default.nix
+        maaf72-go-swagger = with pkgs; buildGoModule rec {
+          pname = "go-swagger-custom";
+          version = "0.30.6-maaf72.1";
+
+          src = fetchGit {
+            # TODO: consider move this fork into efishery repo
+            url = "https://github.com/MAAF72/go-swagger.git";
+            ref = "master";
+            rev = "e05ec5b59149e59afa2acb79479eaec6a68106cf";
+          };
```

Bagian `src` bisa kita manfaatkan `flake: inputs` sehingga gak perlu manage revision dan bisa menggunakan `nix flake lock --update-inputs <ATTR_NAME>` untuk up-to-date dengan input tersebut.

```suggestion
          inputs.go-swagger = {
            url = "github:MAAF72/go-swagger";
            flake = false;
          };
          ......
          src = go-swagger;
```